src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -55,6 +55,18 @@ public function isGranted(mixed $role, mixed $object = null, ?string $field = nu
         }
     }
 
+    public function accessDecision(mixed $role, mixed $object = null, ?string $field = null): AccessDecision
+    {
+        if (!class_exists(AccessDecision::class)) {
+            throw new \LogicException(\sprintf('Using the "access_decision()" function requires symfony/security-core >= 7.3. Try running "composer %s symfony/security-core".', $this->securityChecker ? 'bump' : 'require'));
```

Comment:
TIL `composer bump`

Reply:
Actually, `bump` would not help here as it does not update the package but only modifies the `composer.json` file.

Reply:
To clarify I would update this part as follows:

```suggestion
            throw new \LogicException(\sprintf('Using the "access_decision()" function requires symfony/security-core >= 7.3. Try running "composer %s symfony/security-core".', $this->securityChecker ? 'update' : 'require'));
```